cfme/tests/services/test_orchestration_template.py

```diff
@@ -85,6 +85,8 @@ def test_name_required_error_validation(provisioning, create_template):
                                      description="my template")
     with error.expected(flash_msg):
         template.create(create_template)
+    appliance.browser.quit_browser()
```

Comment:
**REQUIRED:** I'm not sure what you are putting this in for but I don't believe we should have it there.

Reply:
@psav I can remove this, it's just a work around step to run multiple test case with positive outcome, there seems to be an error while clicking the cancel button at the end of the test case, causing issue. I tried to fix that, it seems to be an issue with selenium but still not sure on that.